Lib/typing.py

```diff
@@ -117,17 +121,16 @@ def _type_check(arg, msg):
         return type(None)
     if isinstance(arg, str):
         return ForwardRef(arg)
-    if (
-        isinstance(arg, _GenericAlias) and
-        arg.__origin__ in (Generic, _Protocol, ClassVar) or
-        arg in (Generic, _Protocol)
-        or isinstance(arg, _SpecialForm) and arg is not Any
-    ):
-        raise TypeError("Plain %s is not valid as type argument" % arg)
+    if (isinstance(arg, _GenericAlias) and
+            arg.__origin__ in (Generic, _Protocol, ClassVar)):
+        raise TypeError(f"{arg} is not valid as type argument")
+    if (isinstance(arg, _SpecialForm) and arg is not Any
+            or arg in (Generic, _Protocol)):
+        raise TypeError(f"Plain {arg} is not valid as type argument")
     if isinstance(arg, (type, TypeVar, ForwardRef)):
         return arg
     if not callable(arg):
-        raise TypeError(msg + " Got %.100r." % (arg,))
+        raise TypeError(f"{msg} Got {arg!r:.100}")
```

Comment:
There was a period at the end of the last sentence.

Reply:
This is intentional. I compared with other errors, and this is the only one where there is a dot at the end.

Reply:
Yes, but in this case the error message contains two sentences. For example: "ClassVar accepts only single type. Got (1, 2)".

Reply:
OK, I will return the dot.